application/actions/app.go

```diff
@@ -96,6 +96,8 @@ func App() *buffalo.App {
 
 		app.GET("/conversations", usersThreads)
 
+		app.GET("/requests", getRequests)
```

Comment:
I think it'd be better to rename the handler method to `requestsList` so the resource comes first and the action second

Reply:
I might agree with that, but it's not how we did it in TF. So much standardization work to do, uff-da!